### PR TITLE
Edit and resend email

### PR DESCRIPTION
**Pull request by dwhenry**
Allow correction and resend of emails if they have
been input incorrectly.

### CODE REVIEW

File: spec/models/appointment_summary_spec.rb [hunk 1]
```diff
@@ -169,4 +169,27 @@
...
+    subject { create(:appointment_summary, requested_digital: true, email: 'test@test.com') }
+    before { subject.update(notification_id: '123') }
+
+    it 'resets the notification ID if the email changes' do
+      expect do
+        subject.update(email: 'new@test.com')
+      end.to change { subject.notification_id }.from('123').to('')
+    end
+
+    it 'does not resets the notification ID if the email doesnt changes' do
```
Comment (benbarnett) [inline spec/models/appointment_summary_spec.rb]: `does not reset the notification ID if the email address stays the same`?

File: spec/models/appointment_summary_spec.rb [hunk 2]
```diff
@@ -169,4 +169,27 @@
...
+      expect do
+        subject.update(email: subject.email)
+      end.not_to change { subject.notification_id }
+    end
+
+    it 'does not resets the notification ID if other fields changes' do
```
Comment (benbarnett) [inline spec/models/appointment_summary_spec.rb]: `does not reset the notification ID if fields other than the email address are changed`?

File: features/email_notification.feature [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Feature: Email notification
+  As a Pension Wise customer who has had a phone appointment
```
Comment (benbarnett) [inline features/email_notification.feature]: I think this scenario could be cleaned up a bit...

The subject is "as a Pension Wise customer" but the other steps are referring to the customer as "them" and "I" seems to be the pension wise administrator.

File: app/views/admin/appointment_summaries/index.html.erb [hunk 4]
```diff
@@ -47,6 +48,7 @@
         <td><%= summary.date_of_appointment.to_s(:gov_uk) %></td>
         <td><%= summary.requested_digital ? 'Yes' : 'No' %></td>
         <td><%= summary.email %></td>
+        <td><%= link_to 'Edit email', edit_admin_appointment_summary_path(summary), class: 't-edit-email' %></td>
```
Comment (benbarnett) [inline app/views/admin/appointment_summaries/index.html.erb]: Do we need to put a conditional in here to only show this button if the customer chose a digital appointment summary?

**Comment by dwhenry** [comment]
@benbarnett Add changes made as discussed.

**Comment by benbarnett** [comment]
Nice one 👍

File: app/views/admin/appointment_summaries/edit.html.erb [hunk 5]
```diff
@@ -0,0 +1,53 @@
+<%= form_for @appointment_summary, url: admin_appointment_summary_path(@appointment_summary) do |f| %>
+  <div class="page-header">
+    <h1><%= title 'Resend appointment summary email' %></h1>
+  </div>
+
```
Comment (benlovell) [inline app/views/admin/appointment_summaries/edit.html.erb]: Whoopsies.

File: features/email_notification.feature [hunk 6]
```diff
@@ -0,0 +1,17 @@
+Feature: Email notification
+  As a Pension Wise employee
+  When the customer selected a digital summary document
+  I want to ensure the customer receives a notification email
+
```
Comment (benlovell) [inline features/email_notification.feature]: Whoopsies 😄

**Comment by benlovell** [comment]
Solid 💯, just a few dud whitespace lines for the chop.

**Pull request merged**